interface/src/components/requests.js

```diff
@@ -48,7 +48,7 @@ export function getGeoJSON(file){
 
     formData.append("shp_file", file);
 
-    let res= fetch("http://localhost:3001/getGeoJSON", {
+    let res= fetch("http://localhost:80/getGeoJSON", {
```

Comment:
Same as above, get URI from runtime environment. Here and below

Reply:
Thanks for pointing out! Will change this.